filesystem/01-devices.post

```diff
@@ -1,31 +1,6 @@
 maybe_create_devs ()
 {
   local DEVDIR=/dev
-  # Check for ${DEVDIR} directory
-  if [ -e "${DEVDIR}" -a ! -d "${DEVDIR}" ]
-  then 
-    # No mercy.  Try to remove.
-    rm -f "${DEVDIR}"
-    if [ -e "${DEVDIR}" -a ! -d "${DEVDIR}" ]
-    then 
-      echo
-      echo "${DEVDIR} is existant but not a directory."
-      echo "Please fix that manually, otherwise you WILL get problems."
-      echo
-      exit 1
-    fi
-  fi
-
-  # Create it if necessary
-  mkdir -m 755 "${DEVDIR}" 2> /dev/null
-  if [ ! -e "${DEVDIR}" ]
-  then
-    echo
-    echo "Creating ${DEVDIR} directory failed."
-    echo "Please fix that manually, otherwise you WILL get problems."
-    echo
-    exit 1
-  fi
 
   # Check for ${DEVDIR}/shm directory (for POSIX semaphores and POSIX shared mem)
```

Comment:
Do we need the other things at all?

Reply:
yes, named semaphores and shared memory need those directories from what I see. I was planing to add some integration tests for that.

Not sure what pacman would do if we add those empty dirs to "filesystem". I'll try.

Reply:
I've added two integration tests which fail if the directories are missing:

* https://github.com/msys2/msys2-tests/pull/29
* https://github.com/msys2/msys2-tests/pull/30

checking if they exist is fast since there are no forks involved, so I'd like to defer that to a future cleanup.